main/main.cpp

```diff
@@ -1677,10 +1677,17 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 	OS::get_singleton()->_allow_hidpi = GLOBAL_DEF("display/window/dpi/allow_hidpi", true);
 	OS::get_singleton()->_allow_layered = GLOBAL_DEF("display/window/per_pixel_transparency/allowed", false);
 
+#ifdef TOOLS_ENABLED
 	if (editor || project_manager) {
-		// The editor and project manager always detect and use hiDPI if needed
+		// The editor and project manager always detect and use hiDPI if needed.
 		OS::get_singleton()->_allow_hidpi = true;
+		// Disable MangoHud in editor, as it's applied to all windows.
+		OS::get_singleton()->set_environment("DISABLE_MANGOHUD", "1");
```

Comment:
If we are force disabling tools like this, I guess we should also add `DISABLE_RTSS_LAYER = 1` to fix https://github.com/godotengine/godot/issues/57937

Reply:
RTSS is a part of afterburner.